roles/pulp_webserver/tasks/generate_tls_certificates.yml

```diff
@@ -0,0 +1,69 @@
+---
+- name: Set python-cryptography package name
+  set_fact:
+    python_cryptography: python-cryptography
+  when: ansible_python.version.major == 2
+
+- name: Set python-cryptography package name
+  set_fact:
+    python_cryptography: python3-cryptography
+  when: ansible_python.version.major == 3
+
+- name: Ensure python-cryptography is installed
+  become: true
+  package:
+    name: '{{ python_cryptography }}'
+
+- name: Generate CA key
+  openssl_privatekey:
+    path: '{{ pulp_webserver_tls_folder }}/root.key'
+
+- name: Generate CA CSR
+  openssl_csr:
+    path: '{{ pulp_webserver_tls_folder }}/root.csr'
+    privatekey_path: '{{ pulp_webserver_tls_folder }}/root.key'
+    common_name: www.redhat.com
```

Comment:
Do we want to use "real" names here? I'd suspected to read something like "snakeoil GmBH".
Bonus points: Move the values to default.yaml.

Reply:
Actually I think this needs to be `{{ ansible_fqdn }}` of the hostname of the Pulp installation itself. For example in my PR I was using this instead:  https://github.com/pulp/pulp_installer/pull/254/files#diff-75f75b1315c3a9f242c93c90ee45f1c0R39

Even though the cert is self-signed, that is a trust level problem with the cert, but the cert itself should still be as accurate as possible.